src/DIRAC/FrameworkSystem/Service/TokenManagerHandler.py

```diff
@@ -241,9 +241,30 @@ def export_getToken(
                 # Let's try to revoke broken token
                 idpObj.revokeToken(token["refresh_token"])
 
+        # No token in the cache, try getting
+        kwargs = {"userName": userName,
+                  "timeLeft": requiredTimeLeft,
+                  "group": userGroup,
+                  "scope": scope,
+                  "audience": audience,
+                  }
+        result = idpObj.getToken( **kwargs )
```

Comment:
seems like `idpObj.getToken` returns an `S_OK/S_ERROR` structure, while `idpObj.getGroupScopes` does not. It feels inconsistent.

Reply:
The two methods are far different in complexity. getGroupScopes() would only return S_OK and never S_ERROR. In this case we usually do not return the structure